scripts/snippets_csv/index.js

```diff
@@ -0,0 +1,119 @@
+const ObjectsToCsv = require('objects-to-csv');
+const glob = require('glob-promise');
+const search = require('search-in-file');
+const { gitlogPromise } = require('gitlog');
+const path = require('path');
+const fs = require('fs');
+
+const repoPath = path.join(__dirname, '../../');
+const directories = ['__glossary', '_cci2', '_cci2_ja'];
+
+const log = (message) => {
+  // eslint-disable-next-line no-console
+  console.log('=>', message);
+};
+
+const addToData = async (filePath, lineStart, lineStop) => {
+  let info = {
+    file: filePath,
+    lines: `${lineStart}-${lineStop}`,
+    snippetSize: lineStop - lineStart - 1,
+  };
+
+  const logData = await gitlogPromise({
+    repo: repoPath,
+    fileLineRange: {
+      file: filePath,
+      startLine: lineStart,
+      endLine: lineStop,
+    },
+    number: 100,
```

Comment:
what is `number` for here?

Reply:
And why 100?

Reply:
This is the max amount of commits we want to get out of `git log` by default it's `10` which is not enough for us. 100 seemed a good number for now and the future given that the most amount of commits a snippet has today is in the 40s.